dockerize.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+IMAGE_NAME="boresha/agritrader"
+DOCKER_HOME="/agritrader"
+ARG_NUM=2
+
+if [ "$1" == "init" ]; then
+	docker build -t $IMAGE_NAME $DIR
+elif [ "$1" == "lint" ]; then
+	docker run --rm -v $DIR:$DOCKER_HOME -t \
+		$IMAGE_NAME tslint -c tslint.json 'src/**/*.ts'
```

Comment:
# Requested Change

I think it is a better practice to create a `npm` script for this instead. Again this way we have a global linting script that all developers can use, not just those developing using docker. If the `npm` script calls `node_modules/.bin/tslint ...` this would ensure it is performing linting using the exact version listed in the lock file, not any that are installed globally.

Reply:
Yea, much more pragmatic way of using docker anyways, Must've been been asleep when I made this :P